### PR TITLE
Translation of Web UI

### PR DESCRIPTION
**Pull request by hacketiwack**
PR related to Translation of the Web UI #1458

With this PR, the interface has been entirely (hopefully) translated.
However, the language setting is not yet stored because the underlying API does not provide yet a way to store the language. Therefore, this part should be done in another PR.

### CODE REVIEW

File: web-src/vite.config.js [hunk 1]
```diff
@@ -1,19 +1,37 @@
...
 // Support for setting the OwnTone server URL as env var VITE_OWNTONE_URL
 // in development mode.
 // E. g. start the DEV server with
 //
-//     VITE_OWNTONE_URL=https://owntone.local:3689 npm run serve
+//     VITE_OWNTONE_URL=https://owntone.local:3689; npm run serve
 //
 // will connect the web interface with a remote OwnTone server.
-const owntoneUrl = process.env.VITE_OWNTONE_URL ?? 'http://localhost:3689'
+const owntoneUrl = process.env.VITE_OWNTONE_URL ?? 'http://192.168.0.9:3689'
```
Comment (hacketiwack) [inline web-src/vite.config.js:13]: This is a mistake. It should remain as it was.
```suggestion
const owntoneUrl = process.env.VITE_OWNTONE_URL ?? 'http://localhost:3689'
```

**Review by chme — CHANGES_REQUESTED** [review]
Thanks a lot for implementing this feature!

Due to the many formatting changes, the individual files are difficult to review.
Could you please fix the formatting? Running the following command should be enough to fix all of them:

``` bash
npm run format
```

And please drop the changed `index.js` from this PR. (I will do a rebuilt of the web interface after this is merged).

File: web-src/package.json [hunk 2]
```diff
@@ -31,12 +34,15 @@
     "vuex": "^4.0.2"
   },
   "devDependencies": {
+    "@intlify/vite-plugin-vue-i18n": "^3.4.0",
+    "@intlify/vue-i18n-loader": "^3.0.0",
```
[the remark this answers is not in the document]
Comment (hacketiwack) [reply web-src/package.json]: Agree. Corrected. in the last push.

File: web-src/package.json [hunk 3]
```diff
@@ -2,17 +2,19 @@
...
+    "dev": "vite",
+    "format": "prettier .  --write",
+    "i18n:report": "vue-cli-service i18n:report --src \"./src/**/*.?(js|vue)\" --locales \"./src/locales/**/*.json\"",
+    "preview": "vite preview"
   },
   "dependencies": {
     "@aacassandra/vue3-progressbar": "^1.0.3",
     "@mdi/js": "^6.6.96",
     "@ts-pro/vue-eternal-loading": "^1.2.0",
+    "@vue/cli-shared-utils": "^5.0.4",
```
[the remark this answers is not in the document]
Comment (hacketiwack) [reply web-src/package.json]: Agree. Dropped with the latest commit.

File: web-src/package.json [hunk 4]
```diff
@@ -31,12 +34,15 @@
...
+    "@intlify/vue-i18n-loader": "^3.0.0",
     "@vitejs/plugin-vue": "^2.3.1",
     "eslint": "^8.13.0",
     "eslint-config-prettier": "^8.5.0",
     "eslint-plugin-vue": "^8.6.0",
     "prettier": "2.6.2",
     "sass": "^1.50.0",
-    "vite": "^2.9.5"
+    "vite": "^2.9.8",
+    "vue-cli-plugin-i18n": "~2.3.1"
```
[the remark this answers is not in the document]
Comment (hacketiwack) [reply web-src/package.json]: Agree. Library has been removed in the last commit.

**Review by chme — APPROVED** [review]

**Comment by chme** [comment]
Looks good to me. Thanks!

**Pull request merged**